metastore/iceberg-metastore/README.md

```diff
@@ -15,6 +15,10 @@ can be overwritten in `drill-metastore-distrib.conf` or  `drill-metastore-overri
 `drill.metastore.iceberg.location.base_path` and `drill.metastore.iceberg.location.relative_path` -
 indicate Iceberg Metastore base location.
 
+`drill.metastore.expiration.period` - allows to specify period after which Iceberg table outdated
```

Comment:
Options above and below contain also ```iceberg```.  Does absence of the word here mean that it's generic option or it's simply accidentally missed ?